services/sleeper/metadata/metadata.yml

```diff
@@ -0,0 +1,47 @@
+name: sleeper
+key: simcore/services/comp/sleeper
+type: computational
+integration-version: 1.0.0
+version: 2.0.0
+description: A service which awaits for time to pass, used for system validation.
+contact: neagu@itis.swiss
+authors:
+  - name: "Andrei Neagu"
+    email: neagu@itis.swiss
+    affiliation: "IT'IS Foundation"
```

Comment:
shall we add odei as second? is it not an extension of his work ? ;-)

Reply:
Gladly! How would one do that? Is there a tag for it?

Reply:
this is yaml list, just add a new item, ie.
```yaml
   authors:
       - name: "foo"
         email: "bla"
       - name: "bar"
         email: "fee"
```

Reply:
never mind. I did not notice it was an array.